src/store/main.ts

```diff
@@ -2294,6 +2317,51 @@ export class MainStore {
     }
   }
 
+  async getUserFeatureByUuid(uuid: string): Promise<Feature | undefined> {
+    try {
+      if (!uiStore.meInfo) return undefined;
+      const info = uiStore.meInfo;
+      console.log('TribesURL', TribesURL);
+      const r: any = await fetch(`${TribesURL}/features/${uuid}`, {
+        method: 'GET',

```

Comment:
Pull from master, some of these functions are already implemented

Reply:
@elraphty Addressed